examples/notebooks/creating_a_problem.ipynb

```diff
@@ -0,0 +1,194 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "X87NUGPW04py",
+    "outputId": "0d785b07-7cff-4aeb-e60a-4ff5a669afbf"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Note: you may need to restart the kernel to use updated packages.\n",
+      "Note: you may need to restart the kernel to use updated packages.\n"
+     ]
+    }
+   ],
+   "source": [
+    "%pip install --upgrade pip ipywidgets -q\n",
+    "%pip install pybop -q\n",
+    "\n",
+    "import numpy as np\n",
+    "\n",
+    "import pybop\n",
+    "\n",
+    "pybop.PlotlyManager().pio.renderers.default = \"notebook_connected\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "5XU-dMtU04p2"
+   },
+   "source": [
+    "### Creating a Model\n",
+    "\n",
+    "A fitting problem consists of three parts: An experimental dataset that is used for fitting, a model that the dataset should be fit to, and the parameters that should be fit. To obtain a problem we therefore first need to create a model. This has been explained in detail in the creating_a_model example. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "parameter_set = pybop.ParameterSet.pybamm(\"Marquis2019\")\n",
+    "model = pybop.lithium_ion.SPM(parameter_set=parameter_set)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, the parameters. All parameters of the problem are contained in pybop.Parameters(). Each of the variable to fit is defined as a pybop.Parameter() separated by commas inside pybop.Parameters(). In this example, we have two parameters, the positive electrode thickness and the positive particle radius:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "parameters = pybop.Parameters(\n",
+    "    pybop.Parameter(\n",
+    "        \"Positive electrode thickness [m]\",\n",
+    "        prior=pybop.Gaussian(7.56e-05, 0.05e-05),\n",
+    "        bounds=[65e-06, 10e-05],\n",
+    "    ),\n",
+    "    pybop.Parameter(\n",
+    "        \"Positive particle radius [m]\",\n",
+    "        prior=pybop.Gaussian(5.22e-06, 0.05e-06),\n",
+    "        bounds=[2e-06, 9e-06],\n",
+    "    ),\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The third component is the dataset. This could be an experimental dataset or a simulation. In this example, we just use simulation data by running model.predict() for a 1C discharge, similar to the creating_a_problem example. To pass it to the problem, we need to extract the voltage, time and current data from the simulation results and define it as the dataset:"
```

Comment:
```suggestion
    "The third component is the dataset. This could be an experimental dataset or a simulation. In this example, we just use simulation data by running `model.predict()` for a 1C discharge, similar to the [creating_a_model](https://github.com/pybop-team/PyBOP/blob/develop/examples/notebooks/creating_a_model.ipynb) example. To pass it to the problem, we need to extract the voltage, time and current data from the simulation results and define it as the dataset:"
```